kubernetes-itests/pom.xml

```diff
@@ -34,6 +34,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <junit.version>4.12</junit.version>
```

Comment:
why are you forcing junit 4 here? You can refactor it to use junit 5 packages also, right? Or did you face some problems with arquillian? Maybe arquillian is not compatible with Junit 5? Maybe we need to update arquilian also....

Reply:
@rohanKanojia: In integration tests for @RunWith(ArquillianConditionalRunner.class), I replaced RunWith with ExtendWith for JUnit5 but it was showing compatibility error.

Reply:
@rohanKanojia @dev-gaur: Can you look into this when you get the time and provide suggestions? 
The problem I was facing for itests was with @RunWith(ArquillianConditionalRunner.class)

Reply:
@rohanKanojia @dev-gaur @ricardozanini: Can you look into this when you get the time and provide suggestions? 
The problem I was facing for itests was with @RunWith(ArquillianConditionalRunner.class)